acceptance-tests/src/test/java/uk/nhs/digital/ps/test/acceptance/steps/cms/ps/CmsSteps.java

```diff
@@ -101,6 +124,7 @@ private void createPublishedPublication(final Publication publication) throws Th
     }
 
     // Scenario: New Publication screen ========================================================================
+
```

Comment:
Why all the blank lines?

Reply:
Good spot. Lines left in by accident. I'm removing them now.